### ISSUE TITLE
PPCLink -  # Internal error: File :Src:XCoff:XCoffSection.cp; Line 70 ## Assertion failed: HasPAddr(address + size - 1)

### ISSUE THREAD

**Issue by ksherlock** [ksherlock/mpw#40]
```
$ mpw-make Count
  PPCLink -d -c 'MPS ' -t MPST -sym Full Count.c.x "~/mpw/3.5:Libraries:SharedLibraries:"StdCLib "~/mpw/3.5:Libraries:SharedLibraries:"InterfaceLib "~/mpw/3.5:Libraries:PPCLibraries:"StdCRuntime.o "~/mpw/3.5:Libraries:PPCLibraries:"PPCCRuntime.o "~/mpw/3.5:Libraries:PPCLibraries:"PPCToolLibs.o -o Count.ppc

# Internal error: File :Src:XCoff:XCoffSection.cp; Line 70 ## Assertion failed: HasPAddr(address + size - 1)
PPCLink - Execution terminated!
```
(from 3.5 CExamples)

**Comment by ksherlock** [comment]
generated .c.x file has $0d/$0a conversion problems.

**Comment by ksherlock** [comment]
fixed in 3e2c02f7963a6854dd4973e02bb969af4a637831
